x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/roles_empty_prompt.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiButton, EuiLink, EuiEmptyPrompt } from '@elastic/eui';
+
+import { RolesEmptyPrompt } from './roles_empty_prompt';
+
+describe('RolesEmptyPrompt', () => {
+  const onEnable = jest.fn();
+
+  const props = {
+    productName: 'App Search',
+    docsLink: 'http://elastic.co',
+    onEnable,
+  };
+
+  it('renders', () => {
+    const wrapper = shallow(<RolesEmptyPrompt {...props} />);
+
+    expect(wrapper.find(EuiEmptyPrompt)).toHaveLength(1);
+  });
+
+  it('calls method on change', () => {
+    const wrapper = shallow(<RolesEmptyPrompt {...props} />);
+    const prompt = wrapper.find(EuiEmptyPrompt).dive();
+    prompt.find(EuiButton).simulate('click');
+
+    expect(onEnable).toHaveBeenCalled();
+    expect(prompt.find(EuiLink).prop('href')).toEqual(props.docsLink);
```

Comment:
This assertion probably belongs in its own test or up in the `renders` test.

Reply:
https://github.com/elastic/kibana/pull/102826/commits/d2994add496884392ebf68ffe82732d98dde196b